### DISCUSSION TITLE
About my Student Developer Pack

### DISCUSSION THREAD

**Discussion by Geramtz-17** [community/community#140842] [GitHub Education]
### Hi everyone,

Hi everyone, I need some help, I have some problems with my Student Developer Pack, but its 5 days after I received the confirmation and I don´t have the benefits

**Comment by AudreyLin** [comment]
Hi there. I don't know who is responsible for this here on GitHub, but I do know that Major League Hacking works with it and might be able to help you out in their Discord server. [MLH Discord Server](https://discord.gg/mlh).  I hope this helps you find some answers.  Also, I don't know if you have checked the Learn more sections for each of the benefits [here on this page](https://education.github.com/pack), but maybe they have more information there. Also, they might have sent you an email confirmation to check in before receiving your benefits or if they need more information.  I hope this helps.

**Comment by davevad93** [comment]
Hi @Geramtz-17 ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @Geramtz-17 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @Geramtz-17 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268
